infrastructure/assumptions/non-local-ports.sub.window.js

```diff
@@ -8,6 +8,13 @@
 //
 // More tests can be found in `fetch/private-network-access/`.
 
+const kPorts = {
```

Comment:
Nit: since I am also suggesting other changes, can I request dropping the `k` prefix here and below? That's familiar to Chromium C++ contributors, but isn't a common style in WPT.

Reply:
Done.